packages/app-store/alby/lib/PaymentService.ts

```diff
@@ -84,7 +87,7 @@ export class PaymentService implements IAbstractPaymentService {
       }
       return paymentData;
     } catch (error) {
-      log.error("Alby: Payment could not be created", bookingId);
+      log.error("Alby: Payment could not be created", bookingId, JSON.stringify(error));
```

Comment:
We could use `safeStringify` here. It's shorter plus prevents accidental crash in case we stringify a cyclic object